server/app/repository/VersionRepository.java

```diff
@@ -286,6 +286,11 @@ public boolean isDraftProgram(Long programId) {
         .anyMatch(draftProgram -> draftProgram.id.equals(programId));
   }
 
+  public boolean isActiveProgram(Long programId) {
```

Comment:
**suggestion (blocking):** add javadoc e.g. "True if program is a member of the current active version."